Fix typos and grammar

cbrains opened the pull request:
I went through and fixed up as many typos and grammatical mistakes as I could. I tried not to be too invasive with rewriting things, and I'm open to discuss different ways to word stuff. My main objective was to keep the humor, fix the bugs, and try to get closer to the vanilla style.

There's a lot here, so I recommend looking at each commit individually.

Closes #113

Tachyonite commented:
Looks good. Not sure on the spaces after the ellipsis, think I'd prefer to keep those as-is. Adds a bit of clutter to them. Otherwise the rest of the changes look ok.

cbrains commented:
Ok, I think I've reverted the spaces after ellipses thing. There's still a few places where I removed ellipses entirely in favor of just having two sentences rather than run-ons.

I also found a few more typos while doing this, so win-win.
